### PR TITLE
Add July 20, 2023 Cabal Minutes

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
As the title says.  Add the meeting notes for the July 20, 2023 Podman Community Cabal Meeting.

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
@mairin and @cgiradkar PTAL.  Especially the index.ts change and if I missed any other bits.

File: static/data/meetings/notes/2023-07-20/index.md [hunk 1]
```diff
@@ -0,0 +1,277 @@
+# Podman Community Cabal Meeting Notes
+
+Attendees: Aditya Rajan, Anders F Björklund, Ashley Cui, Ed Santiago Munoz, Jake Correnti, Justin Jereza, Lokesh Mandvekar, Martin Jackson, Matt Heon, Miloslav Trmac, Mohan Boddu, Nalin Dahyabhai, Paul Holzinger, Tom Sweeney, Valentin Rothberg
+
+## July 20, 2023 Topics
```
Comment (cgiradkar) [inline static/data/meetings/notes/2023-07-20/index.md:8]: @TomSweeneyRedHat I see the format is somewhat different here than other notes.
e.g. There is a line like
```
June 15, 2023 11:00 a.m. Eastern (UTC-5)
```
Which is used by the code to fetch the date field.
I think a line as above needs to be added here.

File: static/data/meetings/notes/2023-07-20/index.md [hunk 2]
```diff
@@ -0,0 +1,277 @@
+# Podman Community Cabal Meeting Notes
+
+Attendees: Aditya Rajan, Anders F Björklund, Ashley Cui, Ed Santiago Munoz, Jake Correnti, Justin Jereza, Lokesh Mandvekar, Martin Jackson, Matt Heon, Miloslav Trmac, Mohan Boddu, Nalin Dahyabhai, Paul Holzinger, Tom Sweeney, Valentin Rothberg
```
Comment (cgiradkar) [inline static/data/meetings/notes/2023-07-20/index.md]: The markdown format is different than previous notes. This shouldn't change anything, but date field below needs a line to be added.

**Review by cgiradkar — APPROVED** [review]
lgtm

**Pull request merged**